### ISSUE TITLE
pdfmsym package incompatible

### ISSUE THREAD

**Issue by mbertucci47** [latex3/tagging-project#298]
The [pdfmsym](https://www.ctan.org/pkg/pdfmsym) package is incompatible with the math tagging code. The example
```tex
\DocumentMetadata
  {
    lang=en-US,
    pdfversion=2.0,
    pdfstandard=ua-2,
%    testphase={phase-III,math,title,table,firstaid}
  }
\documentclass{article}

\input{pdfmsym}
\pdfmsymsetscalefactor{12}

\title{pdfmsym tagging test}

\begin{document}

text

\end{document}
```
errors with 
```
! You can't use `\unless' before `\relax'.
<to be read again> 
\ifmeasuring@ 
l.739     \@wide@operator{suum}\sum{.52}{.6}
```
If `math` is removed, the error goes away.

**Comment by u-fischer** [comment]
Something that must be loaded with \input is not really a LaTeX package and I'm inclined to put that under no-support @davidcarlisle?

Apart from this it is imho a package bug: They are doing typesetting in the preamble by using \setbox. The error can be avoided by loading amsmath earlier, but naturally this doesn't mean that the math tagging is then good.

**Comment by davidcarlisle** [comment]
I'd be tempted just to delete it, sonce as you say it's not reallty a package. At some future time we may want more types that just class or package, eg pgf librraies or beamer thesmes or, as here general tex input, but for now I'd simply not list it at all.

**Comment by davidcarlisle** [comment]
If it was `type: tex` and `prioity: 9` it could sit in the yaml file waiting for the future to arrive....

**Comment by FrankMittelbach** [comment]
I'm for delete for the reasons I have outlined before. It is not helpful for the people that actually deal with the yaml file and with the test directories.

**Comment by u-fischer** [comment]
> But please keep a separate list of such rejects, for that future time when we need to consider them!

@car222222 we have the issue here, so the info is not lost.

**Comment by car222222** [comment]
I am not sure that having to search through all the issues is practical.

Why is it not good to use a new file, or files, to keep all the information concerning stuff in which Frank has no current interest (but others maybe will soon want to look at)?

**Comment by car222222** [comment]
@FrankMittelbach  OK, so you want to banish to oblivion everything in which you are not currently interested. Or not?  

But can we not keep all the information currently being discovered in a distinct file, or files?  Maybe these should be in a different location.

**Comment by FrankMittelbach** [comment]
@car222222 yes exactly, what else?  If you want to maintain or collect the extra information I'm all for it, but I don't have the bandwidth to do it.

**Comment by mbertucci47** [comment]
Sorry, I only added this because I noticed an [answer](https://tex.stackexchange.com/a/669164/208544) on TeX-SX by egreg that used it, and a few other answers by the package's author, which usually means it will end up in someone's document somewhere. But if it should be removed from the yml file that's fine

**Comment by FrankMittelbach** [comment]
As it is not a LaTeX package and we do not have a category for .tex files I would suggest we leave it out of the yaml tables for now but we can leave this issue open in case we change our minds and for people that search for it under issues.

by the way, I just looked at the arxiv report covering 2 million+ articles and it isn't used there ever so it is clearly not widespread at least not in professional STEM docs

**Comment by FrankMittelbach** [comment]
didn't we say this isn't a package in the first place and therefore doesn't belong in the list? Or if we we keep it in then after all perhaps with "no-support" (and a reason).
